<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableOver</class>
 <widget class="QWidget" name="TableOver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>624</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>624</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>624</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">*
{
	font: 11px &quot;Arial&quot;;
    color: #c2c2c2;
}

QWidget#TableOver {

	image: url(:/textures/game/TableShadow.png);
background-image: url(:/textures/game/carpet.png);
	border-image: url(:/textures/game/TableBorder.png);
}</string>
  </property>
  <widget class="FramedEntity" name="_bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>624</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>624</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>624</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="respondable">
    <bool>false</bool>
   </property>
   <property name="sizeOfFrame">
    <size>
     <width>1000</width>
     <height>624</height>
    </size>
   </property>
  </widget>
  <widget class="EntityWidget" name="layer_gameTable">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>95</y>
     <width>648</width>
     <height>344</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="backgroundColor">
    <color alpha="6">
     <red>0</red>
     <green>111</green>
     <blue>111</blue>
    </color>
   </property>
   <property name="respondable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="EntityWidget" name="layer_middleSpace">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>624</height>
    </rect>
   </property>
  </widget>
  <widget class="UserControls" name="hud" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>624</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">.QWidget {
border: 1px solid red;

background-image: url(:/textures/mockups/player_mask2.png);
}</string>
   </property>
   <property name="mask" stdset="0">
    <pixmap resource="../resources.qrc">:/textures/mockups/player_mask.png</pixmap>
   </property>
   <widget class="EntityWidget" name="hud_low_level">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>494</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1000</width>
      <height>410</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>500</height>
     </size>
    </property>
    <property name="backgroundColor">
     <color alpha="12">
      <red>0</red>
      <green>0</green>
      <blue>121</blue>
     </color>
    </property>
    <property name="shape">
     <bool>true</bool>
    </property>
    <widget class="QFrame" name="sdg">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>40</y>
       <width>200</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">.QFrame {
    border: 2px outset red;
border-radius: 7px;
}
* {
    color: rgba(30, 30, 10, 250);
	font: 750 16px &quot;Plantagenet Cherokee&quot;;
}

QLabel:!disabled {
    background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 121, 40, 255), stop:0.954802 rgba(0, 199, 83, 255), stop:1 rgba(0, 250, 158, 255));
border: 1px ridge rgba(255, 255, 255, 122) ;
}

#label_s:!disabled {
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 101, 121, 255), stop:0.954802 rgba(0, 199, 190, 253), stop:1 rgba(0, 250, 248, 255));
border: 1px ridge rgba(255, 255, 255, 122) ;
}

QLabel:disabled {
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(121, 121, 121, 255), stop:0.954802 rgba(199, 199, 199, 255), stop:1 rgba(250, 250, 250, 255));
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0" columnstretch="0,1,1,1">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="1" colspan="3">
       <widget class="QLabel" name="label_player">
        <property name="styleSheet">
         <string notr="true">QLabel {
 color: white;   
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(34, 43, 49, 255), stop:1 rgba(69, 89, 99, 255));
font-size: 12px;

border-top-right-radius: 5px;
}</string>
        </property>
        <property name="text">
         <string>player</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_s">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="indent">
         <number>8</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_m">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="indent">
         <number>8</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_a">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
    border-bottom-right-radius: 5px;
}</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="indent">
         <number>8</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>36</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
    border-bottom-left-radius: 5px;
    border-top-left-radius: 5px;
	background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(176, 76, 1, 255), stop:0.954802 rgba(254, 182, 76, 255), stop:1 rgba(255, 248, 146, 255));
}</string>
        </property>
        <property name="pixmap">
         <pixmap resource="../resources.qrc">:/textures/game/icon_clock_timebank.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <zorder>_bg</zorder>
  <zorder>layer_middleSpace</zorder>
  <zorder>hud</zorder>
  <zorder>layer_gameTable</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EntityWidget</class>
   <extends>QWidget</extends>
   <header>plugins.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FramedEntity</class>
   <extends>EntityWidget</extends>
   <header>plugins.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UserControls</class>
   <extends>QWidget</extends>
   <header>game/controls/usercontrols.h</header>
   <slots>
    <slot>enableMask()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
 <slots>
  <signal>setBackground(QImage)</signal>
  <slot>backgroundChanged(QImage)</slot>
 </slots>
</ui>
